FEWD/Week02/lesson03/homework/relaxr_starter_code/css/style.css

```diff
@@ -64,59 +65,119 @@ body {
   width: 100%;
   margin: auto;
   height: 500px;
-  display: block;
 }
 
 header {
+  height: 500px;
+}
 
+h1 {
+  color: white;
+  font-weight: bolder;
+  font-size: 50px;
+  padding: 25px;
+  margin-left: 20px;
 }
 
-h3 {
-  background-color: #f0efef
+h2 {
+  color: white;
+  font-size: 40px;
+  font-weight: 400;
+  padding: 75px;
+  margin-left: 250px;
+}
+
+.button-one {
+  font-size: 24px;
+  font-weight: bolder;
+  background-color: #f9e42e;
+  border: 5px solid #f9e42e;
+  width: 200px;
+  text-align: center;
+  padding: 5px;
+  margin: 0 auto;
+  border-radius: 5px;
+  color: #033048;
 }
 
 .benefits {
   background-color: #f0efef;
+  height: 350px;
```

Comment:
As I mentioned in class, for the most part you want to let blocks be the natural height of the content inside of them. You then want to add padding to either the parent or to the children to create the separation between the edge of the parent and the edge of the child that you're looking for.

So for example here you're dealing with some html that looks roughly like this:

```html
<div class="benefits">
  <h3>Benefits</h3>
  <p class="benefits-para">Text goes here</p>
  <ul>...list items here...</ul>
</div>
```

I can see that below, you've added some padding to your `h3` (the child of `.benefits`). If you're taking that route, then you'll want to add `padding-bottom: 50px;` to the `<ul>` (which is the last child of `.benefits`) so that the space between the beginning of `.benefits` and its first child and the space between the end of `benefits` and its last child is the same.

However that's a bit unnecessarily complex. Best to just add the padding to the parent instead of the children, like so:

```css
.benefits {
  background-color: #f0efef;
  padding: 50px;
}

h3 {
  font-weight: bold;
  font-size: 26px;
  color: grey;
  padding-bottom: 20px;
}

.benefits-para {
  font-size: 20px;
  width: 50%;
  color: grey;
  margin: 0;
  padding-bottom: 20px;
}

.benefits ul {
  padding-left: 50px;
}
```

... by adding the padding to the parent you create the separation you want without forcing the height of the element, see the screenshot below of the highlighted `.benefits` div:

![image](https://cloud.githubusercontent.com/assets/5670279/17406643/34a6699a-5a5c-11e6-9ce6-500f9f4c48cf.png)